playstation-plus.js

```diff
@@ -0,0 +1,269 @@
+import { firefox } from "playwright-firefox"; // stealth plugin needs no outdated playwright-extra
+import { authenticator } from "otplib";
+import {
+    datetime,
+    handleSIGINT,
+    html_game_list,
+    jsonDb,
+    notify,
+    prompt,
+    retryOnError,
+} from "./util.js";
+import path from "path";
+import { existsSync, writeFileSync } from "fs";
+import { cfg } from "./config.js";
+
+// ### SETUP
+const URL_CLAIM = "https://www.playstation.com/en-us/ps-plus/whats-new";
```

Comment:
Thanks @OJ7! The load/login I can have a look at without a Playstation Plus subscription I guess?

@Barokai I just tried that e.g. en-de does not work. If we can't list the games for a country without having to change to its language, it will be problematic for text locators.

Reply:
> @Barokai I just tried that e.g. en-de does not work. If we can't list the games for a country without having to change to its language, it will be problematic for text locators.

you're right en-de doesn't work as it is a non-existent locale.
didn't check all the locators to be honest, thought they were only about class and ID selectors, didn't see the text only ones - only way to solve this would be to make a lookup table for the languages and all the translations of text-only locators...

Reply:
I might push an updates version of this this weekend i habe been working on some things already :)